src/algorithms/synthesis/dd_synthesis.cpp

```diff
@@ -0,0 +1,57 @@
+#include "algorithms/synthesis/dd_synthesis.hpp"
+
+#include "dd/FunctionalityConstruction.hpp"
+using namespace dd::literals;
```

Comment:
I don't quite see where this is necessary. Consider removing it.

Reply:
done 👍🏼